src/express/commands/start.js

```diff
@@ -213,23 +213,15 @@ async function eventuallyCleanupPreviousDevnet(ips, devnetType) {
             await runSshCommand(ip, command, maxRetries)
 
             console.log("📍Stopping ganache (if present) on machine " + ip + " ...")
-            command = `tmux send-keys -t matic-cli-ganache:0 'C-c' ENTER || echo 'ganache not running on current machine...'`
-            await runSshCommand(ip, command, maxRetries)
-
-            console.log("📍Killing ganache tmux session (if present) on machine " + ip + " ...")
-            command = `tmux kill-session -t matic-cli-ganache || echo 'matic-cli-ganache tmux session does not exist on current machine...'`
+            command = `sudo systemctl stop ganache.service || echo 'ganache not running on current machine...'`
             await runSshCommand(ip, command, maxRetries)
         }
         console.log("📍Stopping heimdall (if present) on machine " + ip + " ...")
-        let command = `tmux send-keys -t matic-cli:0 'C-c' ENTER || echo 'heimdall not running on current machine...'`
+        let command = `sudo systemctl stop heimdalld.service || echo 'heimdall not running on current machine...'`
         await runSshCommand(ip, command, maxRetries)
 
         console.log("📍Stopping bor (if present) on machine " + ip + " ...")
-        command = `tmux send-keys -t matic-cli:1 'C-c' ENTER || echo 'bor not running on current machine...'`
-        await runSshCommand(ip, command, maxRetries)
-
-        console.log("📍Killing matic-cli tmux session (if present) on machine " + ip + " ...")
-        command = `tmux kill-session -t matic-cli || echo 'matic-cli tmux session does not exist on current machine...'`
+        command = `sudo systemctl stop bor.service || echo 'ganache not running on current machine...'`
```

Comment:
`bor` is not running in this case, not `ganache`

Reply:
Fixed: [9209192](https://github.com/maticnetwork/matic-cli/pull/91/commits/9209192266f222638db82d359e2f5c56f19a793d)